crates/client/src/config.rs

```diff
@@ -94,8 +204,323 @@ impl Config {
         }
         throw!(Error::BadWorkspace)
     }
+    pub fn merge_with_cli(&mut self, cli_vars: &str) {
```

Comment:
Is this even necessary? What if we just prepend the parameters defined in Trdelnik.toml to the HFUZZ_RUN env variable? Or maybe append based on the precedence. But I guess if an argument is defined twice, only the first or the last will be taken into account...